vault/logical_system_namespaces_seals_test.go

```diff
@@ -160,6 +160,101 @@ func TestNamespaceBackend_SealUnseal(t *testing.T) {
 		res, err = b.HandleRequest(nsCtx, req)
 		require.NoError(t, err)
 	})
+
+	t.Run("preserve mounts after unsealing namespaces", func(t *testing.T) {
+		namespace_name := "unseal_preserve_mount"
```

Comment:
doesn't matter, but camelCase is the way
```suggestion
		namespaceName := "unseal_preserve_mount"
```